lib/pagseguro/account.rb

```diff
@@ -0,0 +1,29 @@
+module PagSeguro
+  class Account
+    include Extensions::MassAssignment
+    include Extensions::EnsureType
+
+    # Set the account e-mail.
+    attr_accessor :email
+
+    # Set the account type.
+    # Must be PERSONAL, SELLER or COMPANY.
+    attr_accessor :type
+
+    # Get the person account.
```

Comment:
Yes... some comments I think is unnecessary, but I puts this to keep the standard.